ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot

```diff
@@ -49,22 +49,3 @@ Clear Data Science Pipelines Resources
     Delete Pipeline    ${pipeline_id}
     Remove Pipeline Project    ${project}
```

Comment:
Isn't there an openshiftlibrary keyword to remove a namespace?

Reply:
you could use `Oc Delete    kind=Project   name=${project_k8s_name}`

Reply:
We have keyword ` Oc Delete  kind=Project  name=<name of the project>`

Reply:
Remove Pipeline Project will be addressed in other PR. The current code [double check ](https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/libs/DataSciencePipelinesAPI.py#L84-L97)wait because it could be in Terminating status